operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/ConfigurationServiceOverrider.java

```diff
@@ -121,6 +122,10 @@ public ConfigurationServiceOverrider withWorkflowExecutorService(
   /**
    * Replaces the default {@link KubernetesClient} instance by the specified one. This is the
    * preferred mechanism to configure which client will be used to access the cluster.
+   * <p>
+   * Note that when {@link Operator#stop()} is called, by default the client is closed even if
+   * explicitly provided with this method. Use {@link #withKubernetesClient(KubernetesClient)} to
```

Comment:
This isn't the correct link to the method to use

Reply:
ahh, sorry, fixed